### ISSUE TITLE
V-Sync lock to Screen refresh rate instead of 60 fps?

### ISSUE THREAD

**Issue by metallic77** [weihuoya/citra#392]
Many games run full speed on my cellphone like e.g. Shantae runs constantly on 60 fps but screen actual refresh on my Xperia XZ1 with Snapdragon 835 is 59 so i get a glitch every frame. Isn't it possible to detect refresh rate and sync on that instead?

**Comment by RaduNastase** [comment]
Yeah, this would be great. Very few phones have a perfect 60 refresh rate, and even a 0.5 difference can cause micro stutters.
